ci-operator/config/openshift/vmware-vsphere-csi-driver-operator/openshift-vmware-vsphere-csi-driver-operator-main.yaml

```diff
@@ -123,6 +123,15 @@ tests:
     env:
       TEST_SCENARIOS: \[CSI-Driver\]|\[vSphere CSI Driver\]|\[vSphere-CSI-Driver-Operator\]
     workflow: openshift-e2e-vsphere-csi-extended
+- as: e2e-vsphere-csi-vcf9
+  steps:
+    cluster_profile: vsphere-elastic
+    env:
+      ENABLE_LONG_CSI_CERTIFICATION_TESTS: "true"
```

Comment:
```suggestion
```
We'd better not run the long during tests in presubmit CI, it should be covered in some periodic CI.